kblocks/experiments/core.py

```diff
@@ -86,7 +87,7 @@ def operative_config_path(self) -> str:
         return _operative_config_path(self._experiment_dir)
 
     @property
-    def status(self) -> str:
+    def status(self) -> Union[str, Status]:
```

Comment:
Ah yes, I turned off mypy a while back because it was annoying me - but this should always be a `Status` object.